sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3072,6 +3086,8 @@ class SQLConf extends Serializable with Logging {
 
   def integerGroupingIdEnabled: Boolean = getConf(SQLConf.LEGACY_INTEGER_GROUPING_ID)
 
+  def parquetRebaseDateTimeEnabled: Boolean = getConf(SQLConf.LEGACY_PARQUET_REBASE_DATETIME)
```

Comment:
It's not recommended to add a method to read a conf if it's only used once.

Reply:
It has been already used in 9 places.

Reply:
Yea I know, but we should stop it. Let's follow https://github.com/apache/spark/commit/e736c62764137b2c3af90d2dc8a77e391891200a#diff-d07d3850a0ad8635a0c5562ed1e8bf26R184

Reply:
Is it a correct link?

Reply:
I need to read the config in Java. The obvious replacement doesn't work:
```java
this.rebaseDateTime = SQLConf.get().getConf(SQLConf.LEGACY_PARQUET_REBASE_DATETIME());
```
```
Error:(134, 48) java: incompatible types: inference variable T has incompatible bounds
    equality constraints: java.lang.Object
    upper bounds: java.lang.Boolean,java.lang.Object
```
If you have strong opinion about removing the method `def parquetRebaseDateTimeEnabled`, could you show me how to read a SQL config in Java. I haven't found any example, unfortunately:
```
$ find . -name "*.java" -print0|xargs -0 grep 'SQLConf.get'
./sql/core/src/test/java/test/org/apache/spark/sql/JavaBeanDeserializationSuite.java:      DateTimeUtils.getZoneId(SQLConf.get().sessionLocalTimeZone()));
./sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedColumnReader.java:    this.rebaseDateTime = SQLConf.get().parquetRebaseDateTimeEnabled();
```